app/helpers/view_helper.rb

```diff
@@ -0,0 +1,36 @@
+module ViewHelper
+  def govuk_link_to(body, url = body, html_options = { class: "govuk-link" })
```

Comment:
This is included from `govuk_components` so I don't think we need to define it. Same for `govuk_back_link_to`

Reply:
Removed.